src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -4,88 +4,103 @@ short-title: Integrate Flutter
-the module project that contains some CocoaPods and a helper Ruby script.
+Add your Dart code to the `lib/` directory.
+
+Add Flutter dependencies to `my_flutter/pubspec.yaml`, including Flutter packages
+and plugins.
 
-## Make the host app depend on the Flutter module
+The `.ios/` hidden subfolder contains a Xcode workspace where you can 
+run a stand-alone version of your module, and is a wrapper project to bootstrap
+your Flutter code. It contains helper scripts to build frameworks or
+embed the module into your existing application with [CocoaPods][].
 
-The description below assumes that your existing iOS app has a structure similar
-to what you get by asking Xcode to generate a new "Single View App" project
-using Objective-C. If your existing app has a different folder structure
-and/or existing `.xcconfig` files, you can reuse those, but you'll probably need
-to adjust some of the relative paths mentioned below accordingly.
+{{site.alert.note}}
+Add custom iOS code to your existing application or a plugin, not to
+the module in `.ios/`. Changes made in `.ios/` are not embedded in your existing application.
+Regenerate the directory by running `flutter clean` or `flutter pub get` in the
+`my_flutter` directory.
+{{site.alert.end}}
 
-The assumed folder structure is as follows:
+## Embed the Flutter module in your existing application
+
+There are two ways to embed Flutter in your existing application.
+1. Use the CocoaPods dependency manager and installed Flutter SDK. Recommended.
+1. Create frameworks for the Flutter engine, your compiled Dart code, and all Flutter plugins.
+Manually embed the frameworks and update your existing application's build settings in Xcode.
+
+### Embed with CocoaPods and the Flutter SDK
+
+This method requires every developer working on your project to have a locally installed
+version of the Flutter SDK.
+
+The following example assumes that your existing application and the Flutter module are in sibling directories.
+If you have a different directory organization, you may need to adjust the relative paths.
 
 ```text
 some/path/
   my_flutter/
-    lib/main.dart
-    .ios/
+    .ios/Flutter/podhelper.rb
   MyApp/
-    MyApp/
-      AppDelegate.h
-      AppDelegate.m (or swift)
-      :
+    Podfile
 ```
 
-### Add your Flutter app to your Podfile
-
-Integrating the Flutter framework requires using the CocoaPods dependency
-manager. This is because the Flutter framework needs to be available to any
-Flutter plugins that you might include in `my_flutter`.
-
-For information on how to install CocoaPods on your development machine, see
-[cocoapods.org](https://cocoapods.org/).
-
-If your host application (`MyApp`) is already using CocoaPods, you only have to do the
-following to integrate with your `my_flutter` app:
+If your existing application (`MyApp`) does not already have a Podfile,  follow the
+[CocoaPods getting started guide][] to add a `Podfile` to your project.
 
```

Comment:
Actually I suspect a non-trivial amount of people might fall into the i-opened-the-xcodeproj-file-instead-of-the-xcworkspace-file and will run into an error. Perhaps spell that out as well.

Reply:
I mentioned it.  Fortunately `pod` on its own has a better log about using `xcworkspace` than `flutter` does, so hopefully they see it.